### PR TITLE
feat: use understandable name

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Use variable names that are easy to understand. Sorry, but there is a destructive change to `apm.json`. I wanted to get this in before release.

Since it involves destructive changes, I'm using improvement instead of refactoring.

**This PR relies on #123 .**

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/lib/apmJson.js [hunk 1]
```diff
@@ -138,7 +138,7 @@ module.exports = {
     const apmJson = getApmJson(instPath);
     apmJson.packages[package.id] = {
       id: package.id,
-      repository: package.repo,
+      repository: package.repository,
```
Comment (hal-shu-sato) [inline src/lib/apmJson.js:141]: I think this key name is not changed, so this is not breaking change.
I suggest you change the commit message.

**Review by hal-shu-sato — COMMENTED** [review]
I think it is not breaking change.

File: src/lib/apmJson.js (hunk 1 above)
Comment (hal-shu-sato) [reply src/lib/apmJson.js:141]: If you checked, please resolve this conversation.
(Perhaps, you cannot do this? I don't know about the detail of the collaborator's permission.)
(If you do, let me know.)

Comment (mitosagi) [reply src/lib/apmJson.js:141]: You are certainly right. I hadn't noticed.

**Review by hal-shu-sato — APPROVED** [review]

**Pull request merged**